src/psm/test/corners.ok

```diff
@@ -0,0 +1,34 @@
+[INFO ODB-0222] Reading LEF file: Nangate45/Nangate45.lef
+[INFO ODB-0223]     Created 22 technology layers
+[INFO ODB-0224]     Created 27 technology vias
+[INFO ODB-0225]     Created 135 library cells
+[INFO ODB-0226] Finished LEF file:  Nangate45/Nangate45.lef
+[INFO ODB-0128] Design: gcd
+[INFO ODB-0130]     Created 54 pins.
+[INFO ODB-0131]     Created 624 components and 2752 component-terminals.
+[INFO ODB-0132]     Created 2 special nets and 1248 connections.
+[INFO ODB-0133]     Created 581 nets and 1504 connections.
+[INFO PSM-0001] Reading voltage source file: Vsrc_gcd_vdd.loc.
+[INFO PSM-0015] Reading location of VDD and VSS sources from Vsrc_gcd_vdd.loc.
+[INFO PSM-0076] Setting metal node density to be standard cell height times 5.
+[WARNING PSM-0030] VSRC location at (50.000um, 50.000um) and size 20.000um, is not located on an existing power stripe node. Moving to closest node at (68.070um, 53.115um).
+[INFO PSM-0031] Number of PDN nodes on net VDD = 604.
+[INFO PSM-0064] Number of voltage sources = 1.
+[INFO PSM-0040] All PDN stripes on net VDD are connected.
+########## IR report #################
+Worstcase voltage: 6.12e-01 V
+Average IR drop  : 4.42e-01 V
+Worstcase IR drop: 4.88e-01 V
```

Comment:
I think it would be good if the output indicated the corner it comes from.

Reply:
I can add that, anything else that would be helpful while I'm in there?

Reply:
Nothing comes to mind